backend/internal/common/env_config.go

```diff
@@ -38,6 +39,7 @@ var EnvConfig = &EnvConfigSchema{
 	Host:                     "0.0.0.0",
 	MaxMindLicenseKey:        "",
 	GeoLiteDBPath:            "data/GeoLite2-City.mmdb",
+	GeoLiteDBUrl:             "",
```

Comment:
```suggestion
	GeoLiteDBUrl:             "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=%s&suffix=tar.gz",
```

I would save the default value here.